src/app/config/app-settings.ts

```diff
@@ -12,6 +12,6 @@ export class AppSettings {
     public static HEADER_CONTENT_TYPE = 'application/json';
     public static EMAIL_PATTERN = '[a-z|A-Z|0-9]+[@]+[a-z|A-Z|0-9]+[.]+([a-z|A-Z|0-9]){2}';
     public static USERNAME_REGEX = '/^[a-zA-Z0-9_]{1,15}$/';
-    public static COMPANY_NAME_REGEX = '^[a-zA-Z0-9- ]*$';
+    public static COMPANY_NAME_REGEX = '^([\\w\\d\\s\\-\\,\\./]+)$';
```

Comment:
Consider using this: https://github.com/communitybridge/easycla/blob/master/cla-backend-go/utils/regex.go#L21